Formula/d/dolphie.rb

```diff
@@ -118,7 +148,7 @@ class Dolphie < Formula
   end
 
   def install
-    virtualenv_install_with_resources
+    virtualenv_install_with_resources start_with: "setuptools"
```

Comment:
I don't think we even need the `start_with`? I don't see any issue locally

Reply:
oh, it was a bit preventive. I can try removing it. :)